components/_previews/_preview--body.njk

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Do we need to add this preview to Fractal?  Tried using the default and it still seemed to work "as expected"?

![Uploading image.png…]()

Reply:
the default one adds layout that only will ever be seen in fractal–land - I think it was to show the difference with the full bleed - I'll have to take another look

Reply:
having a look at this -- if we use the defaultt, when we open it 'with layout open' - you can't see the divider because of the lack of top padding.